Source/Diagnostics/ReducedDiags/FieldReduction.H

```diff
@@ -225,15 +226,10 @@ public:
         if (std::is_same_v<ReduceOp, amrex::ReduceOpSum>)
         {
             amrex::ParallelDescriptor::ReduceRealSum(reduce_value);
-        // If reduction operation is a sum, multiply the value by the cell volume so that the
-        // result is the integral of the function over the simulation domain.
-#if defined(WARPX_DIM_1D_Z)
-            reduce_value *= dx[0];
-#elif defined(WARPX_DIM_XZ) || defined(WARPX_DIM_RZ)
-            reduce_value *= dx[0]*dx[1];
-#else
-            reduce_value *= dx[0]*dx[1]*dx[2];
-#endif
+            // If reduction operation is a sum, multiply the value by the cell volume so that the
+            // result is the integral of the function over the simulation domain.
+            const auto cell_volume = std::reduce(std::begin(dx), std::end(dx), 1.0_rt, std::multiplies<>());
+            reduce_value *= cell_volume;
         }
```

Comment:
I've done things like this, which is also compact
```
reduce_value *= AMREX_D_TERM(dx[0], *dx[1], *dx[2]);
```
In fact, I already make this change in PR #5189.

Reply:
Oh, I see... sorry! Then, this PR has no reason to exist. I will close it immediately!